hail/src/main/scala/is/hail/io/bgen/LoadBgen.scala

```diff
@@ -450,7 +451,7 @@ case class MatrixBGENReader(
       EntriesWithFields(includeGT, includeGP, includeDosage),
       dropCols,
       RowFields(includeLid, includeRsid, includeOffset, includeFileIdx),
-      referenceGenome,
+      referenceGenome.map(_.broadcast),
       indexAnnotationType)
 
 //    assert(tr.typ == settings.matrixType)
```

Comment:
Should this be deleted?

Reply:
Yes, those aren't even compatible types anymore.